File: Mock/Data/Exception/CanNotFindDataException.cs
namespace Mock.Data.Exception
{
    /// <summary>
    /// 找不到数据异常
    /// </summary>
    public class CanNotFindDataException : TaistException
    {
        /// <summary>
        /// 构造找不到数据异常的新实例
        /// </summary>
        public CanNotFindDataException(string name) : base(string.Format("Can not find data {0}", name)) { }
    }
}

File: java/Taist.Framwork/src/Mock/Data/CanNotFindDataException.java
package Mock.Data;

import Mock.Exception.TaistException;

public class CanNotFindDataException extends TaistException {
	/**
	 * 
	 */
	private static final long serialVersionUID = 1476845347854588957L;

	public CanNotFindDataException(String message) {
		super(String.format("Can not find data because %s", message));
	}
}
